### PR TITLE
Bug 1140613 - Import every element to the shadow root r=wilsonpage

### PR DESCRIPTION
**Pull request by julienw**

### CODE REVIEW

**Comment by wilsonpage** [comment]
It would be useful if you could add an example to the `index.html` to show how people can add extra stuff to the header bar :)

**Comment by julienw** [comment]
I can do this :)

**Comment by julienw** [comment]
@wilsonpage done !

File: index.html [hunk 1]
```diff
@@ -23,6 +23,12 @@
       margin: 1.5rem 3rem;
     }
 
+    .indicator {
+      background:
+        url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAeCAYAAAA7MK6iAAAAtElEQVRIx+3SoQrCUBSA4YFgEUTMVk2+hdHoY/gAYtkLTIPFaLZY9Om+ZjmWW6a4Ow33h8PYYexj41ZVqVT6hzD/CYpH7zj2Ae/7RGe4B3zHrA90il2gr9lh2jW0QI0zbgmYzi2eq7H4Fp7Eyx4fzBmTLr56jNOb6AnjLn/5CE0L2mCU42AtW+BlrhO9boHXueBtAl2T+20u+BjABSsM4nqJ/TEXfMAGw2Q/jP2hKpVKpdw9AcDkjaObIxshAAAAAElFTkSuQmCC) no-repeat center / 3rem;
```
Comment (wilsonpage) [inline index.html]: I couldn't see this in the default 'settiings' theme. Using `data-icon="star-full"` might work better :snowman:

Comment (julienw) [inline index.html]: done !

**Pull request merged**